packages/fast-components-styles-msft/src/outline-button/index.ts

```diff
@@ -65,18 +64,13 @@ const styles: ComponentStyles<LightweightButtonClassNameContract, DesignSystem>
             ...highContrastDisabledBorder,
         },
         ...highContrastOutline,
-        "a&": {
+        "a&:not($button__disabled)": {
```

Comment:
Something about this nesting looks odd. It duplicates the `not` portion and also includes a selector for when it _is_ disabled. Is this working correctly?

Reply:
yes it is, Kham experimented removing the nested not selector and ran into the specificity issues again because we are specifically targeting `a&:not($button__disabled):hover` simply having `:hover` here gets overwrote.